ia-wishlist-bot/fetch_bookcovers.py

```diff
@@ -0,0 +1,76 @@
+"""
+Python Script to fetch Bookcovers from 
+1. Google Books API
+2. Amazon Scraping Bookcovers
+3. Betterworld Books Page
+"""
+
+import argparse
+import requests
+import json
+from bs4 import BeautifulSoup
+from random import choice
+
+# Random Desktop Agents to avoid detection
```

Comment:
This comment sounds suspicious. What is the intent here? Hopefully nothing illegal or immoral.

Reply:
No nothing illegal, basically a lot of websites block `request.get` if they don't have any headers! They also block if the same header keeps hitting regularly. That's why I've randomised the headers.